src/test/skript/tests/syntaxes/expressions/ExprNewBannerPattern.sk

```diff
@@ -0,0 +1,36 @@
+test "new banner pattern":
+	set {_pattern} to a creeper banner pattern colored red
+	# Will fail due to the "banner pattern" item alias in skript-aliases
```

Comment:
you should make a pr to remove it

Reply:
I thought Skript-Aliases was being removed for 2.10? Since this will be added for that

Reply:
The aliases will be disabled by default for new configs, the actual deprecation should start in 2.11

Reply:
ahh ok